src/eth_provider/constant.rs

```diff
@@ -18,7 +14,7 @@ use {
     },
     starknet_crypto::FieldElement,
     std::sync::Arc,
-    std::{env::var, sync::OnceLock},
+    std::{env::var, str::FromStr, sync::OnceLock},
```

Comment:
do we need this FromStr?

Reply:
Yes because of the `FieldElement::from_str`, if you check it was actually there before